ReactQt/runtime/src/logger.cpp

```diff
@@ -0,0 +1,76 @@
+
+/**
+ * Copyright (c) 2017-present, Status Research and Development GmbH.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include "logger.h"
+#include <QBuffer>
+
+const QString LOG_SETTINGS_FILE = "logger_settings.ini";
```

Comment:
As we discussed, introduction of another one settings file (similar to .env) should create additional complexity to manage settings for status-react and others derived projects.
Probably, for future, what if to embed support of react-native-config and .env files directly into react-native-desktop on native level.